src/cards/top-languages-card.js

```diff
@@ -317,7 +317,6 @@ const renderTopLanguages = (topLangs, options = {}) => {
 
   let finalLayout = "";
   if (layout === "compact" || hide_progress == true) {
-    width = width + 50; // padding
     height =
```

Comment:
I'm not very familiar with SVG, so could you please explain this for me? You seem to just delete this line, so what was the point of this line in the first place? Any idea why this was written this way? Thinking about that will help us figure out if changing this will break anything else. Thanks for the PR!

Reply:
I can't be extremely certain why that line was written like that but the most probable guess would be.
The author wanted the `MIN_CARD_WIDTH` to be `230px` but the UI was breaking (as shown in the 3rd screenshot). Thus adding more `50px` to accommodate for the padding.

Anyhow that line is the root cause of the bug. So, I removed that line and increase the `MIN_CARD_WIDTH` by `50`.  Which will fix the case when the card is too small.

Reply:
Got it; thanks for the explanation!